src/Tabs/Tabs.js

```diff
@@ -1,111 +1,129 @@
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
-import { BrowserRouter, Link } from 'react-router-dom';
 import React, { Component } from 'react';
 
-export const Tabs = props => {
-    const { children, className, ...rest } = props;
+export const Tab = props => {
+    const { children, className, content, disabled, id, onClick, selectedId, tabContentProps, tabLinkProps, url, ...rest } = props;
 
-    const tabClasses = classnames(
-        'fd-tabs-container',
-        className
-    );
+    const renderLink = () => {
+        if (url) {
+            return (
+                <a
+                    {...tabLinkProps}
+                    aria-disabled={disabled}
+                    className={className}
```

Comment:
Does the `className` prop go here or should that go on the `<li>`?  If they really wanted a `className` on the `<a>` or `Link`, they could pass it through `tabLinkProps`.

Reply:
This functionality was not present in the last version of tabs so lets put it on the `<li>` and if a user needs it that can add it to `tabLinkProps` to get it to the anchor.